tutorials/quickstart-python.md

```diff
@@ -370,9 +368,7 @@ tutorials:
 - [Migrate your own Data][migrate]
 
 
-[timescale-cloud]: https://www.timescale.com/products
-[timescale-cloud-install]: /getting-started/installation/timescale-cloud/installation-timescale-cloud
-[timescale-community-install]: /getting-started/installation
+[install-timescale]: /getting-started/installation
```

Comment:
Not an error, but I noticed a missing link ref at line 50.  Can you put in a reference for the [pg-libpq-string] link?